wgengine/router/ifconfig_windows.go

```diff
@@ -662,13 +662,26 @@ func getInterfaceRoutes(ifc *winipcfg.IPAdapterAddresses, family winipcfg.Addres
 // syncRoutes incrementally sets multiples routes on an interface.
 // This avoids a full ifc.FlushRoutes call.
 func syncRoutes(ifc *winipcfg.IPAdapterAddresses, want []*winipcfg.RouteData) error {
-	routes, err := getInterfaceRoutes(ifc, windows.AF_INET)
+	routes4, err := getInterfaceRoutes(ifc, windows.AF_INET)
 	if err != nil {
 		return err
 	}
 
-	got := make([]*winipcfg.RouteData, 0, len(routes))
-	for _, r := range routes {
+	routes6, err := getInterfaceRoutes(ifc, windows.AF_INET6)
+	if err != nil {
+		// TODO: what if v6 unavailable?
```

Comment:
I don't think you want to return the error here. If IPv6 is off, I think this'll just fail and it'll break IPv4-only hosts, making syncRoutes return early here.

Reply:
Everywhere else, we error out if v6 returns errors. Filing a bug to figure out if we need to work harder, or if windows always returns something even if users push the "turn IPv6 off" buttons.